### PR TITLE
[DOC] Review integration page

### PR DESCRIPTION
**Pull request by lucabelluccini**
Clarifications about Ansible/Puppet support.

FYI @elasticdog @nkammah

### CODE REVIEW

**Review by elasticdog — APPROVED** [review]
LGTM

File: docs/plugins/integrations.asciidoc [hunk 1]
```diff
@@ -82,18 +82,13 @@ releases 2.0 and later do not support rivers.
 [[deployment]]
 === Deployment
 
-[discrete]
-==== Supported by Elasticsearch:
-
+==== Supported by the community:
 * https://github.com/elastic/ansible-elasticsearch[Ansible playbook for Elasticsearch]:
```
Comment (elasticdog) [inline docs/plugins/integrations.asciidoc:86]:
```suggestion
* https://github.com/elastic/ansible-elasticsearch[Ansible]:
```

**Review by elasticdog — COMMENTED** [review]
...actually, I did have one suggestion to make the list more consistent.

File: docs/plugins/integrations.asciidoc [hunk 2]
```diff
@@ -82,18 +82,13 @@ releases 2.0 and later do not support rivers.
 [[deployment]]
 === Deployment
 
-[discrete]
-==== Supported by Elasticsearch:
-
+==== Supported by the community:
```
Comment (jrodewig) [inline docs/plugins/integrations.asciidoc:85]:
```suggestion
[discrete]
==== Supported by the community:
```

**Review by jrodewig — APPROVED** [review]
LGTM. I'm going to incorporate the suggestions. One is required to pass the docs CI.

Thanks @lucabelluccini!

**Pull request merged**